tests/performance/config/perf-driver/policies/test-continuous-n-apps.yml

```diff
@@ -66,6 +66,7 @@ channels:
     deploy:
       - type: app
         repeat: "{{apps}}"
+        parallel: 100
```

Comment:
Did you try it? Does it produce better run time?

Reply:
> Did you try it?

Yup

> Does it produce better run time?

Nop 😭